ui/components/app/confirm/info/row/text.tsx

```diff
@@ -57,9 +63,12 @@ export const ConfirmInfoRowText: React.FC<ConfirmInfoRowTextProps> = ({
           className={editIconClassName || undefined}
           color={IconColor.primaryDefault}
           ariaLabel={t('edit')}
-          marginLeft={4}
           iconName={IconName.Edit}
           onClick={onEditClick}
+          size={ButtonIconSize.Sm}
+          // to reset the button padding
+          style={{ marginLeft: '-4px' }}
+          data-testid="edit-nonce-icon"
```

Comment:
I hope this left margin does not effect existing use cases.

Reply:
I made the change in the name of consistency with the designs, so I think it's safe.